src/coverlet.core/CoverageDetails.cs

```diff
@@ -6,6 +6,13 @@ public class CoverageDetails
     {
         public double Covered { get; internal set; }
         public int Total { get; internal set; }
+        private double averageModulePercent;
+        public double AverageModulePercent
+        {
+            get { return Math.Round(averageModulePercent, 2); ; }
```

Comment:
nit: double token `;`

Reply:
Also, can you do `return Math.Floor(averageModulePercent * 100)/100;`

To keep consistency with the way Coverlet handles rounding. We round down using floor. You can refer `Percent `property below this line.

Reply:
Done